cpg_workflows/jobs/joint_genotyping.py

```diff
@@ -486,7 +486,7 @@ def add_make_sitesonly_job(
     j.image(image_path('gatk'))
     res = STANDARD.set_resources(j, ncpu=2)
     if storage_gb:
-        j.storage(storage_gb)
+        j.storage(f'{storage_gb}Gi')
```

Comment:
You're specifying Gi, but the code suggests you provide it it in `Gb`. I don't mind, but would be good to standardise this to Gi (my preferred), or specify `G`, eg:

```suggestion
        j.storage(f'{storage_gb}G')
```